CHANGELOG.md

```diff
@@ -819,7 +840,9 @@ Install with `npm install onfido-sdk-ui@0.12.0-rc.1`
 
 - NPM (commonjs2) style of importing the library now works
 
-[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/6.2.0...development
+[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/6.3.0...development
+[6.2.1]: https://github.com/onfido/onfido-sdk-ui/compare/6.2.1...6.3.0
```

Comment:
shouldn't this be 6.3.0? looks like there are two entries for 6.2.1

Reply:
great spot. will edit that.